### ISSUE TITLE
Generate entities from database tables are failing  for JPA 3.x

### ISSUE THREAD

**Issue by lak-proddev** [eclipse-dali/webtools.dali#10]
Generate entities from database tables are failing for JPA 3. x due to the package name changes. https://github.com/eclipse-dali/webtools.dali/issues/6#issuecomment-2107726547

> _In JPA 3.x the package name changed from "javax.persistence" to "jakarta.persistence".
> We need to make these changes in code generation._

**Comment by nitind** [comment]
It would be better to vary the package named used based on 1) the facet version, if present or 2) which version of the APIs are on the project build path, rather than a hard switchover to always using the Jakarta namespace.

**Comment by lak-proddev** [comment]
> It would be better to vary the package named used based on 1) the facet version, if present or 2) which version of the APIs are on the project build path, rather than a hard switchover to always using the Jakarta namespace.

Agree. That's the reason for delaying the fix. Otherwise, it's just a simple constant value change.

**Comment by nitind** [comment]
@lak-proddev 3.34 is tagged, I'll soon be switching our builds over to 3.35.

**Comment by lak-proddev** [comment]
It should be fixed with JPA 3.x support.
